stable/admin/templates/statefulset.yaml

```diff
@@ -27,8 +27,15 @@ spec:
   template:
     metadata:
       name: {{ template "admin.fullname" . }}
-      {{- if .Values.admin.podAnnotations }}
       annotations:
+      {{- if .Values.admin.configFiles }}
+        {{- with .Values.admin.configFiles }}
+        checksum/config: {{ toYaml . | sha256sum }}
```

Comment:
it checksums the configmap file created by the complete list of config files. Yes a change in order of Values.database.configFiles will cause a change in checksum.   I don't know if it's a big deal,  it would just imply more rolling upgrade than needed.

Reply:
I'm just wondering if the order of entries is identical to what is specified in the values file, or if they are arbitrarily reordered. `configFiles` is specified as a map instead of a list, so it seems possible that they are reordered, generating an upgrade when there is no change at all in the values file. Maybe this can be avoided by sorting the entries somehow.